activation/handler.go

```diff
@@ -322,9 +321,7 @@ func (h *Handler) StoreAtx(ctx context.Context, atx *types.VerifiedActivationTx)
 						Data: &atxProof,
 					},
 				}
-				if proofBytes, err := codec.Encode(proof); err != nil {
-					return err
-				} else if err = identities.SetMalicious(tx, atx.NodeID(), proofBytes); err != nil {
+				if err = h.cdb.AddMalfeasanceProof(atx.NodeID(), proof, dbtx); err != nil {
 					return err
```

Comment:
Please, consider adding some context so we know what operation failed:
```suggestion
					return fmt.Errorf("adding malfeasense proof: %w", err)
```